client/analytics/ad-tracking.js

```diff
@@ -1,57 +1,54 @@
 /**
  * External dependencies
  */
-const async = require( 'async' ),
-	noop = require( 'lodash/utility/noop' ),
-	map = require( 'lodash/collection/map' ),
-	some = require( 'lodash/collection/some' ),
-	debug = require( 'debug' )( 'calypso:ad-tracking' );
+import async from 'async';
+import noop from 'lodash/utility/noop';
+import map from 'lodash/collection/map';
+import some from 'lodash/collection/some';
+import debugFactory from 'debug';
```

Comment:
If we're refactoring this, what do you think about making the order alphabetical too?